docs/adrs/v2-setup-dotnet.md

```diff
@@ -0,0 +1,23 @@
+# V2 setup-dotnet
+
+Date: 2021-02-15
+Status: Proposed
+
+# Context
+- GitHub-hosted Ubuntu and Windows runners have .NET versions pre-installed in system directories, but the `v1` version of action installs .NET to user's directory for that runners. It means that action always download and install .NET version even if it is pre-installed and after using the action all pre-installed .NET versions are unavailable.
```

Comment:
Please, also specify the exact directories: system directory with pre-installed versions and directory that is used by action